File: eBibliotekaClient/src/app/data/interfaces/Author.ts
import { Library } from "./Library";

export class Author{
    id: number;
    name: string;
    /**
     *
     */
    constructor() {
        this.id = -1;
        this.name = "Učitavanje";
    }
}

export interface AuthorUpdateVM{
    id: number;
    name: string;
}

File: eBibliotekaServer/BookModule/Models/Author.cs
using eBibliotekaServer.LibraryModule.Models;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace eBibliotekaServer.BookModule.Models
{
    public class Author
    {
        [Key]
        public int ID { get; set; }
        [Required]
        public string Name { get; set; }

        public int LibraryID { get; set; }
        [ForeignKey(nameof(LibraryID))]
        public Library Library { get; set; }
    }
}
